downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-eda-set-up-new-decision-environment"]
+
+= Setting up a new decision environment
+
+Decision environments are a container image to run Ansible rulebooks.
+They create a common language for communicating automation dependencies, and provide a standard way to build and distribute the automation environment.
+The default decision environment is found at link:https://quay.io/repository/ansible/ansible-rulebook[Ansible-Rulebook]. 
+
+To create your own decision environment refer to link:https://access.redhat.com/solutions/7018085[How do I Build a Custom Decision Environment for Event Driven Ansible within Ansible Automation Platform?].
+
+The following steps describe how to import a decision environment into your {EDAcontroller} Dashboard.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials] section
+of the {ControllerNameStart} documentation.
+* You have pushed a decision environment image to an image repository or you chose to use the image `de-supported` provided at link:http://registry.redhat.io/[registry.redhat.io].
+
+== Procedure
+
+. Navigate to the {EDAcontroller} Dashboard.
+. From the navigation bar, select btn:[Decision Environments] → btn:[Create decision environment].
```

Comment:
I believe we need to call this the navigation "panel". Also,change these buttons to menu items as follows:

select menu:Decision Environments[Create decision environment].